js/actions/appActions.js

```diff
@@ -1292,6 +1292,19 @@ const appActions = {
     AppDispatcher.dispatch({
       actionType: appConstants.APP_DRAG_CANCELLED
     })
+  },
+
+  /**
+   * Notifies autoplay has been blocked
+   * @param {string} location - Location of current frame
+   * @param {number} tabId - Tab id of current frame
+   */
+  autoplayBlocked: function (location, tabId) {
+    AppDispatcher.dispatch({
+      actionType: appConstants.APP_AUTOPLAY_BLOCKED,
```

Comment:
I just mean that the browser process already has this info so it is not needed to send from the window renderer. Pls do for a follow up but not high priority.